README.md

```diff
@@ -3,30 +3,45 @@
 We use this repository as a place to keep code we use across our apollo implementing services' repositories.
 These include plugins, dataloader functions, etc.
 
+## Development
+
+1. `npm install`
+2. `npm run test` run tests or `npm run test:watch` to watch files and run tests when the code changes.
+3. Be sure to export any new modules in `src/index.ts`
+
+## Dependency
+
+1. Update the apollo server version to 3 and node version to 16 to import this package.
+
 ## Example Usage
 
 #### batchCacheFn
```

Comment:
We might just want to remove `batchCacheFn` from the README. It's marked for deprecation already. Sorry for not doing that before.

Reply:
no worries can do

Reply:
done